### ISSUE TITLE
Boolean value false doesn't match

### ISSUE THREAD

**Issue by brianvanburken** [mcollina/bloomrun#38]
I've found this weird case when matching `Boolean` values. I am using the latest version `2.1.0`. This is either intended (but not documented) or a bug.

The following code demonstrates the problem:
```javascript
'use strict'

var bloomrun = require('bloomrun')()

bloomrun.add({ say: true }, 'Match true!');
bloomrun.add({ say: false }, 'Match false!');
bloomrun.default('Global');

console.log( bloomrun.lookup({say: true }) );
console.log( bloomrun.lookup({say: false }) );
console.log( bloomrun.lookup({hello: 'world'}) );
```

Running this code outputs the following:
```bash
$ node index.js
Match true!
Default
Default
```

The expected output is:
```bash
$ node index.js
Match true!
Match false!
Default
```

**Comment by mcollina** [comment]
Thanks for reporting, it's fixed now... I'll be publishing a release out soon.

**Comment by mcollina** [comment]
Released as v2.2.1.

**Comment by brianvanburken** [comment]
Thank you very much! ❤️ 💚 💙 💛 💜
